utils/constants.py

```diff
@@ -1,2 +1,2 @@
-MAX_TILE_ZOOM = 15  # 15 tilezen's max supported zoom
```

Comment:
We play some tricks here with max zoom and tile sizes. For 256 tiles I recall the max zoom is 16, but for 512 tiles it's effectively 15, but when we build 256 tiles and 512 tiles we send the precut 256 zoom 16 tile for the 512 zoom 15 request. It's unclear to me what _this_ `MAX_TILE_ZOOM` is referring to and what it means in the context of a 512 px tile size only build. Do you know? Otherwise I'll do some reading...

Reply:
> We play some tricks here with max zoom and tile sizes. For 256 tiles I recall the max zoom is 16, but for 512 tiles it's effectively 15, but when we build 256 tiles and 512 tiles we send the precut 256 zoom 16 tile for the 512 zoom 15 request. It's unclear to me what _this_ `MAX_TILE_ZOOM` is referring to and what it means in the context of a 512 px tile size only build. Do you know? Otherwise I'll do some reading...

The variable defined here is for bbox build, it iterate all the zooms up to MAX_TILE_ZOOM to find tiles that touches the bounding box.

Reply:
Cool, LGTM